src/ComboControls.ts

```diff
@@ -254,7 +259,10 @@ export default class ComboControls extends EventDispatcher {
     y = (y / domElement.clientHeight) * -2 + 1;
 
     const dollyIn = delta < 0;
-    this.dolly(x, y, this.getDollyDeltaDistance(dollyIn, Math.abs(delta)));
+    const deltaDistance = this.camera instanceof PerspectiveCamera ?
+      this.getDollyDeltaDistance(dollyIn, Math.abs(delta)) :
+      Math.sign(delta) * 0.3;
```

Comment:
Why `0.3`?

Reply:
It's a "magic number" I tried that work the best

Reply:
We should not have any magic numbers. We should expose it as config